drivers/adc/adc_stm32.c

```diff
@@ -656,6 +656,15 @@ static int start_read(const struct device *dev,
 	 * This register controls the analog switch integrated in the IO level.
 	 */
 	LL_ADC_SetChannelPreSelection(adc, channel);
+#elif defined(CONFIG_SOC_SERIES_STM32U5X)
+	/*
+	 * Each channel in the sequence must be previously enabled in PCSEL.
+	 * This register controls the analog switch integrated in the IO level.
+	 * Only for ADC1 instance of the stm32U5.
```

Comment:
Can you provide more info this comment to explain this is limited to ADC1 ?

Reply:
explicitely mentionning that ADC4 has not that register